docs/hugo/content/introduction/annotations.md

```diff
@@ -0,0 +1,27 @@
+# Annotations understood by the operator
+
+## Annotations specified by the user
+
+Note that unless otherwise specified, allowed values are _case sensitive_ and should be provided in lower case.
```

Comment:
Maybe we apply Postel's law and accept with a case-insensitive match?

Reply:
I believe users will expect these to be case sensitive as that is the behavior elsewhere. For example see:
[Kubernetes labels are case sensitive](https://downey.io/notes/dev/kubernetes-labels-are-case-sensitive/).

Or from the documentation [names in Kubernetes are case sensitive](https://kubernetes.io/docs/reference/kubectl/_print/)
> NAME: Specifies the name of the resource. Names are case-sensitive. 

Since the type used to store the annotations by k8s is `map[string]string`, the key is by definition case sensitive. If we tried to make the keys case insensitive we introduce the possibility of multiple keys differing only by case but with conflicting values. We could make the value case insensitive but I argue that it is the name of a "mode" and so should match the label/name casing rules and also be case sensitive.

Reply:
I'm not convinced, but I'll go with whatever you choose. 

Failing to do what the user wanted because they typed `Skip` instead of `skip` seems needlessly pedantic.